--- tora-3.2/cmake/modules/FindQScintilla.cmake~	2017-07-04 23:35:39.000000000 +0200
+++ tora-3.2/cmake/modules/FindQScintilla.cmake	2022-07-16 08:17:16.878361755 +0200
@@ -84,7 +84,7 @@
     SET(LIBRARYPATH ${QT5_WIDGETSLIBRARYPATH} "/usr/lib/" "/usr/local/lib")
     MESSAGE("QScintilla2 LIBPATH \"${LIBRARYPATH}\"")
 
-    FIND_LIBRARY(QSCINTILLA_LIBRARY NAMES qscintilla2-qt5 qt5scintilla2 libqscintilla2.so libqscintilla2.a qscintilla2.lib PATHS ${LIBRARYPATH})
+    FIND_LIBRARY(QSCINTILLA_LIBRARY NAMES qscintilla2_qt5 qscintilla2-qt5 qt5scintilla2 libqscintilla2.so libqscintilla2.a qscintilla2.lib PATHS ${LIBRARYPATH})
 
     # Check
     IF(QSCINTILLA_LIBRARY)
