diff -x 'moc_*' -ur tora-2.0.0/src/todebug.cpp tora-2.0.0-gcc/src/todebug.cpp
--- tora-2.0.0/src/todebug.cpp	2008-09-14 23:54:40.000000000 +0200
+++ tora-2.0.0-gcc/src/todebug.cpp	2009-09-04 13:08:05.743468734 +0200
@@ -79,6 +79,7 @@
 #include <QMessageBox>
 #include <QButtonGroup>
 #include <QTreeView>
+#include <cstdio>
 
 #include "icons/addwatch.xpm"
 #include "icons/changewatch.xpm"
diff -x 'moc_*' -ur tora-2.0.0/src/tomodeleditor.cpp tora-2.0.0-gcc/src/tomodeleditor.cpp
--- tora-2.0.0/src/tomodeleditor.cpp	2008-09-22 02:34:08.000000000 +0200
+++ tora-2.0.0-gcc/src/tomodeleditor.cpp	2009-09-04 13:06:45.023161704 +0200
@@ -58,6 +58,7 @@
 #include <QDesktopWidget>
 #include <QSettings>
 #include <QAbstractItemModel>
+#include <cstdio>
 
 #include "icons/commit.xpm"
 #include "icons/copy.xpm"
diff -x 'moc_*' -ur tora-2.0.0/src/toqvalue.cpp tora-2.0.0-gcc/src/toqvalue.cpp
--- tora-2.0.0/src/toqvalue.cpp	2008-09-14 23:54:40.000000000 +0200
+++ tora-2.0.0-gcc/src/toqvalue.cpp	2009-09-04 13:05:45.063279462 +0200
@@ -46,6 +46,7 @@
 
 #include <QVariant>
 #include <QApplication>
+#include <cstdio>
 
 static int NumberFormat;
 static int NumberDecimals;
diff -x 'moc_*' -ur tora-2.0.0/src/toresultview.cpp tora-2.0.0-gcc/src/toresultview.cpp
--- tora-2.0.0/src/toresultview.cpp	2008-09-14 23:54:40.000000000 +0200
+++ tora-2.0.0-gcc/src/toresultview.cpp	2009-09-04 13:07:33.740202733 +0200
@@ -75,6 +75,7 @@
 #include <QAction>
 #include <QMimeData>
 #include <QColorGroup>
+#include <cstdio>
 
 static int MaxColDisp;
 static bool Gridlines;
