diff -ur tora-2.0.0-gcc/config/m4/postgresql.m4 tora-2.0.0/config/m4/postgresql.m4
--- tora-2.0.0-gcc/config/m4/postgresql.m4	2008-11-03 21:32:20.000000000 +0100
+++ tora-2.0.0/config/m4/postgresql.m4	2009-09-04 13:40:05.626436760 +0200
@@ -6,7 +6,7 @@
   AC_CHECK_DECL([PQcancel],
                 [ac_pg_cancel=yes],
                 [ac_pg_cancel=no],
-                [#include <postgresql/libpq-fe.h>])
+                [#include <libpq-fe.h>])
 
   if test $ac_pg_cancel = yes; then
     AC_DEFINE([LIBPQ_DECL_CANCEL], 1, [Define if libpg-fe.h declares PQcancel.])
diff -ur tora-2.0.0-gcc/configure.ac tora-2.0.0/configure.ac
--- tora-2.0.0-gcc/configure.ac	2008-09-14 23:54:41.000000000 +0200
+++ tora-2.0.0/configure.ac	2009-09-04 13:37:15.446425066 +0200
@@ -28,7 +28,7 @@
 fi
 # Checks for header files.
 # AC_HEADER_STDC
-AC_CHECK_HEADERS(unistd.h sys/param.h sys/time.h time.h sys/mkdev.h sys/sysmacros.h string.h memory.h fcntl.h dirent.h sys/ndir.h ndir.h alloca.h locale.h utime.h postgresql/libpq-fe.h)
+AC_CHECK_HEADERS(unistd.h sys/param.h sys/time.h time.h sys/mkdev.h sys/sysmacros.h string.h memory.h fcntl.h dirent.h sys/ndir.h ndir.h alloca.h locale.h utime.h libpq-fe.h)
 AC_HEADER_MAJOR
 AC_FUNC_ALLOCA
 AC_FUNC_MALLOC
diff -ur tora-2.0.0-gcc/src/toqsqlconnection.cpp tora-2.0.0/src/toqsqlconnection.cpp
--- tora-2.0.0-gcc/src/toqsqlconnection.cpp	2008-09-14 23:54:40.000000000 +0200
+++ tora-2.0.0/src/toqsqlconnection.cpp	2009-09-04 13:39:41.136783352 +0200
@@ -67,7 +67,7 @@
 #include <QGridLayout>
 #include <QString>
 
-#ifdef HAVE_POSTGRESQL_LIBPQ_FE_H
+#ifdef HAVE_LIBPQ_FE_H
 #ifdef CMAKE_BUILD
   // There is different location of pgsql includes sometimes
   // E.g. /usr/include/pgsql on suse etc. Cmake hadles it
@@ -75,7 +75,7 @@
   #include <libpq-fe.h>
 #else
   // This is for autotools
-  #include <postgresql/libpq-fe.h>
+  #include <libpq-fe.h>
 #endif
 #endif
 
