--- tora-2.1.2/config/m4/oracle.m4~	2009-10-23 18:19:57.000000000 +0200
+++ tora-2.1.2/config/m4/oracle.m4	2010-09-11 01:11:18.232423513 +0200
@@ -212,6 +212,7 @@
 #include <oci.h>
 #include <iostream>
 #include <fstream>
+#include <cstdlib>
 using namespace std;
 
 int main(int c, char **v) {
--- tora-2.1.2/src/tooracleconnection.cpp~	2010-03-21 21:09:32.000000000 +0100
+++ tora-2.1.2/src/tooracleconnection.cpp	2010-09-11 01:23:39.588423501 +0200
@@ -898,7 +898,7 @@
             {
                 // Ignore any errors here
             }
-            return QString::QString();
+            return QString();
         }
 
         virtual toQuery::queryImpl *createQuery(toQuery *query, toConnectionSub *sub)
